api/src/pcapi/core/subscription/api.py

```diff
@@ -93,7 +93,9 @@ def _send_beneficiary_activation_email(user: users_models.User, has_activated_ac
 
 
 def has_completed_profile(user: users_models.User) -> bool:
-    return user.city is not None and user.activity is not None
+    return (
+        user.city is not None and user.activity is not None and user.firstName is not None and user.lastName is not None
```

Comment:
Je trouve ça pas très esthétique de tout mettre en vrac sur une ligne, mais en même temps je n'ai pas particulièrement d'alternative. 🙃

Reply:
"suggestion de présentation" :  :)

```suggestion
        all((user.city, user.activity, user.firstName, user.lastName))
```

**Attention :** ça ne distingue pas `None` des chaînes vides, je ne sais pas si c'est raisonnable ici ou pas !

Reply:
> ça ne distingue pas None des chaînes vides, je ne sais pas si c'est raisonnable ici ou pas !
Ah
Je check avec mon PO